### PR TITLE
Start

### PR DESCRIPTION
**Pull request by maksym-khavelov**

### CODE REVIEW

File: src/db/models/AuthorModel.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class AuthorModel {
-  constructor(authorName) {
-    this.name = authorName;
+  constructor(name) {
+    this.name = name;
```
Comment (maksym-khavelov) [inline src/db/models/AuthorModel.js]: Add abstract class

**Review by maksym-khavelov — COMMENTED** [review]
Move entities to schema
Move models to entities
Services works with repositories and entities

File: src/routes/mainRoute.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
```
Comment (maksym-khavelov) [inline src/routes/mainRoute.js:1]: change to static middleware

**Review by maksym-khavelov — COMMENTED** [review]
Rename Client to Base

File: src/entities/AuthorModel.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 class AuthorModel {
-  constructor(authorName) {
-    this.name = authorName;
+  constructor(name) {
```
Comment (vadym-dolzhenko) [inline src/entities/AuthorModel.js:4]: You need to create a base class with the name property

**Review by vadym-dolzhenko — CHANGES_REQUESTED** [review]

**Review by vadym-dolzhenko — APPROVED** [review]

**Pull request merged**